deploy/crds/kieapp.crd.yaml

```diff
@@ -342,6 +342,62 @@ spec:
                   console:
                     description: ConsoleObject configuration of the RHPAM workbench
                     properties:
+                      cors:
+                        description: CORS Cross Origin Resource Sharing configuration
+                          to be used by the KieAPP for Kieserver, BC and Dashbuilder
```

Comment:
```suggestion
                          to be used by the KieApp for the RHPAM workbench
```

Reply:
The description is picked from kieapp_types and is only one, it generate the description in the kieapp.crd.yaml where is used the CORS type